modules/order/tests/src/Kernel/Entity/OrderTest.php

```diff
@@ -22,10 +23,11 @@ class OrderTest extends KernelTestBase {
    * @var array
    */
   public static $modules = ['system', 'field', 'options', 'user', 'entity',
```

Comment:
We should move the first line as well then ("'system', 'field', 'options', 'user', 'entity',")